File: src/main/java/frc/team4069/robot/commands/ElevatorToggleCommand.java
package frc.team4069.robot.commands;

public class ElevatorToggleCommand extends CommandBase {

    @Override
    protected void initialize() {
        if(elevator.isStarted()) {
            elevator.stop();
        }else {
            elevator.start();
        }
    }

    @Override
    protected boolean isFinished() {
        return true;
    }
}

File: src/main/java/frc/team4069/robot/commands/ElevatorToggleCommand.kt
package frc.team4069.robot.commands

import edu.wpi.first.wpilibj.command.Command
import edu.wpi.first.wpilibj.command.ConditionalCommand
import frc.team4069.robot.Robot

class ElevatorToggleCommand(onTrue: Command,
                            onFalse: Command) : ConditionalCommand(onTrue, onFalse) {

    override fun condition(): Boolean = Robot.elevator.isStarted

    override fun isFinished(): Boolean {
        return true
    }
}
